src/pages/playlist/playlist.css

```diff
@@ -0,0 +1,96 @@
+
+.playlist-main{
+    padding: 1rem 5rem;
+    display: grid;
+    gap: 1rem;
+}
+
+.playlist-header, .cp-header{
+    align-items: center;
+}
+.create-playlist-button{
+    justify-self: flex-start;    
+}
+
+.create-playlist-button:hover{
+    background-color: var(--bg-danger);
+    color: var(--text-primary)
+}
+
+.playlist:nth-child(even){
+    cursor: pointer;
+    background-color: var(--bg-highlight);
+}
+
+.playlist:nth-child(odd){
+    cursor: pointer;
+    background-color: var(--bg-warning);
+}
+
+.playlist:hover{
+    background-color: var(--text-primary);
+    color: var(--bg-primary);
+}
+
+.playlist-content-details{
+    font-size: 0.8rem;
+    font-weight: 700;
+    color: var(--text-secondary);
+}
+
+.delete-playlist{
+    align-self: center;
+    padding: 1em;
+    cursor: pointer;
+}
+
+.create-playlist-layer{
+    position: absolute;
+    background-color:#00000096; 
+    width: 100%;
+    height: 100vh;
+    color: var(--white);
+    /* visibility: hidden;this class should be replaced by overlay layer class from component library, once it is updated there.  */
```

Comment:
Not sure if this comment is intentional but just want to point it out to in case its unnecessary and you forgot to remove it

Reply:
Code looks good to merge